opencl-cpu/Readme.txt

```diff
@@ -0,0 +1 @@
+This folder is for OpenCL CPU runtime.
```

Comment:
Better to create the `opencl-cpu` folder inside `opencl` root folder to avoid making more folders on the root level.

Reply:
we create opencl-cpu under llvm following the same layout as internal repo.  It will be more convenient for opencl cpu development.